aldryn_people/boilerplates/bootstrap3/templates/aldryn_people/includes/person.html

```diff
@@ -51,7 +51,7 @@
         <div class="lead">{{ person.description|safe }}</div>
     {% endif %}
 
-    {% if person.vcard_enabled %}
+    {% if not instance and person.vcard_enabled or instance.show_vcard and person.vcard_enabled %}
```

Comment:
I think parentheses would make this much easier to read. I had to read twice to understand what's going on.